flow/project.py

```diff
@@ -3345,25 +3440,23 @@ def key_func_by_job(operation):
             # Optionally re-order operations for execution if order argument is provided:
             if callable(order):
                 operations = list(sorted(operations, key=order))
-            elif order == "random":
-                random.shuffle(operations)
-            elif order in ("by-job", "cyclic"):
+            elif order == "cyclic":
```

Comment:
This ordering code has been mostly reverted to its pre-aggregation form. by-op was removed, and the rest should look like what is on `master`.